pkg/client/restclient/client.go

```diff
@@ -205,6 +205,16 @@ func createSerializers(config ContentConfig) (*Serializers, error) {
 	return s, nil
 }
 
+func (c *RESTClient) ManipulateVersionedAPIPath(fn func(string) string) *RESTClient {
```

Comment:
@ymqytw could you make versionedAPIPath public and summarize why we need the hack, then ask Clayton if it's acceptable.